tests/bluetooth/iso/bsim/tests/2RTN_100ms_ACL.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+SCRIPT_DIR=$(dirname "$(realpath "${BASH_SOURCE[0]}")")
+SCRIPT_NAME=$(basename "$0")
+
+${SCRIPT_DIR}/../iso_and_acl.sh ${SCRIPT_NAME} \
```

Comment:
Rather than calling that script with these arguments, have you considered defining a BASH function in that file and `source` it instead? 

One approach is not necessarily better than the other, but worth considering

Reply:
Discuss